.github/workflows/validate-packs.yml

```diff
@@ -0,0 +1,40 @@
+name: Validate Packs
+
+on:
+  pull_request:
+    paths:
+      - "packs/**"
+      - "ci/**"
+      - ".github/workflows/validate-packs.yml"
+
+jobs:
+  validate_packs:
+    name: Validate
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+
+      - name: Get changed files
+        # https://github.com/hanseltimeindustries/get-changed-files
+        uses: hanseltimeindustries/get-changed-files/@0b9ab9b0bdbe9384ab0202b4c119fd8f88747335 # v1.1.2
+        id: changed_files
```

Comment:
Is really expensive to run? Any reason we wouldn't just run it over the whole thing?